src/profile_manager/templates/profile_manager/form.html

```diff
@@ -22,10 +23,21 @@
         </div>
     </div>
 
+
 <form action="" enctype="multipart/form-data" method="post">{% csrf_token %}
   <a href="{% url 'profiles:profile_detail' profile.id %}" role="button" class="btn btn-danger">Cancel</a>
   <input type="submit" value="{{ submit_btn_value }}" class="btn btn-success" /><br><br>
-  {{ form|crispy }}
+
+  {% if request.user.is_staff %}
+    <p>Want to change <a href="{% url 'profiles:change_password' profile.user.pk %}">password</a>?</p>
+  {% endif %}
+
+
+  {% for form in forms %}
+    {{ form|crispy }}
```

Comment:
Nice.  However, I would like an H3 before the user form at the bottom. Something like this (though form1 and form2 are not what they'll be called... I don't know actually!  You can probably just use [django template filter](https://docs.djangoproject.com/en/4.0/ref/templates/builtins/#for) `forloop.counter` to see if you are in the second part of the loop to add the h3 and second form

{{ form1|crispy }}
<h3>Authentication and Permissions</h3>
<p>Only staff members may edit these fields.</p>
{{ form2|crispy}}

Reply:
done